content/applications/websites/livechat/reports.rst

```diff
@@ -0,0 +1,176 @@
+- :guilabel:`Average duration`: the average duration of a conversation, in seconds.
+- :guilabel:`Average rating`: the average rating received by the operator.
+- :guilabel:`Time to answer`: the average amount of time before the operator responds to a chat
+  request.
+- :guilabel:`Count`: the total number of sessions.
+
+.. image:: reports/operator-analysis.png
+   :align: center
+   :alt: Example of the Operator Analysis report from the Live Chat application.
+
+View and filter options
+=======================
+
+On any Odoo report, the view and filter options vary, depending on what data is being analyzed,
+measured, and grouped. See below for additional information on the available views for the *Live
+Chat* reports.
+
+.. note::
+   The :ref:`Sessions History <livechat/sessions-history>` report is **only** available in *list
+   view*.
+
+Pivot view
+----------
+
+The *pivot* view presents data in an interactive manner. The :ref:`Session Statistics
+<livechat/session-statistics>` and :ref:`Operator Analysis <livechat/operator-analysis>` reports are
+available in pivot view.
+
+The pivot view can be accessed on a report by selecting the :guilabel:`grid icon` at the top-right
+of the screen.
+
+To add a group to a row or column to the pivot view, click the :guilabel:`➕ (plus sign)` next to
+:guilabel:`Total`, and then select one of the groups. To remove one, click the :guilabel:`➖ (minus
+sign)` and de-select the appropriate option.
+
+Graph view
+----------
+
+The *graph* view presents data in either a *bar*, *line*, or *pie* chart. The :ref:`Session
+Statistics <livechat/session-statistics>` and :ref:`Operator Analysis <livechat/operator-analysis>`
+reports are available in graph view.
+
+Switch to the graph view by selecting the :guilabel:`line chart icon` at the top-right of the
+screen. To switch between the different charts, select the *related icon* at the top-left of the
+chart, while in graph view.
+
+.. tip::
+   Both the bar chart and line chart can utilize the stacked view option. This presents two or more
+   groups of data on top of each other, instead of next to each other, making it easier to compare
+   data.
+
+Save and share a favorite search
+--------------------------------
+
+The *Favorites* feature found on reports allows users to save their most commonly used filters
+without having to reconstruct them every time they are needed.
+
+To create and save a *Favorites* on a report, follow the steps below:
+
+#. Set the necessary parameters using the :guilabel:`Filters`, :guilabel:`Group By` and
+   :guilabel:`Measures` options.
+#. Click the :guilabel:`🔻(triangle pointed down)` icon next to the :guilabel:`Search...` bar to
+   open the drop-down menu.
+#. Under the :guilabel:`Favorites` heading, click :guilabel:`Save current search`.
+#. Rename the search.
+#. Select :guilabel:`Use by default` to have these filter settings automatically displayed when the
+   report is opened. Otherwise, leave it blank.
```

Comment:
From what I am seeing, you click the option- so there's nothing to leave blank, you just don't select it (unless I am not looking at my runbot the right way)